src/marqo/tensor_search/on_start_script.py

```diff
@@ -28,6 +34,7 @@ def on_start(config: config.Config):
         SetBestAvailableDevice(),
         ModelsForCacheing(),
         InitializeRedis("localhost", 6379),
+        PrewarmPatchModels(),
```

Comment:
Can you name this so it's more consistent with `ModelsForCacheing`? Maybe call it `CachePatchModels` and rename the other one to `CacheModels`.

Reply:
changed